doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -962,6 +962,17 @@ HomeKit
 
 The issues in this section are related to the HomeKit protocol.
 
+.. rst-class:: v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+
+KRKNWK-17343: Accessories becoming signaling slower when some data pair in the NVS changes frequently.
+  Access to data pairs that change rarely can take a lot of time.
+  This problem manifests itself for example, in case of long booting or paired/unpaired identify response time.
+  Due to this issue, there may be problems with certification, because during certification tests accessory is burdened with a huge amount of resets, pair/unpair procces resulting in a large number of new data pairs stored in NVS.
+  As a result, the accesory does not pass the certification test cases validating maximum operation time e.g. TCT012, TCT022 and TCT023.
```

Comment:
```suggestion
KRKNWK-17343: Accessory's signal is slower when data pairs in the NVS change frequently
  Accessing data pairs that rarely change can take a long time, which happens, for example, during long booting or a paired/unpaired identify process.
  Because of this, user might experience certification issues, as during the certification, the accessory is subjected to multiple resets and pair/unpair processes, which causes NVS to store a large amount of new data pairs.
  As a result, the accessory does not pass the certification test cases exceeding the maximum operation time (for example, TCT012, TCT022 and TCT023).
```
Not sure if I understood the last part correctly - plz check